dom/practice/js/scripts.js

```diff
@@ -0,0 +1,39 @@
+// 1: Set the text of the <h1> element
+const headline = document.querySelector('h1');
+headline.textContent = `My Activities List`;
+
+// 2: Set the color of the <h1> to a different color
+headline.style.color = 'skyblue';
+
+// 3: Set the content of the '.desc' paragraph
+// The content should include at least one HTML tag
+const desc = document.querySelector('.desc');
+desc.innerHTML = `<strong>A list of <i>fun</i> things I want to do today.</strong>`;
+
+// 4: Set the class of the <ul> to 'list'
+const list = document.querySelector('ul');
+list.className = 'list';
+
+// 5: Create a new list item and add it to the <ul>
+
+list.insertAdjacentHTML(
+    'beforeend',
+    `<li><input> Eat Oreos</li>`
+);
+// 6: Change all <input> elements from text fields to checkboxes
+
```

Comment:
I know its a nitpick but your spacing is off here - empty line after 5. No empty line before 6.

It doesn't matter now but when it does matter it can hurt. 
Make sure your looking at the code not just for syntax errors but for overall style consistency too.

Reply:
Done